projects/MarlQ/proposal/README.md

```diff
@@ -0,0 +1,56 @@
+# Proposal: (Volume-preserving) Soft Object Denting
+
+MarlQ (Marcel Quanz)
+
+01.01.2022
+
+## Abstract
+
+The basic idea of the project is to simulate the volume displacement of a soft (squishy) object by a hard object.
+In practice, an actor, e.g. a hand, displaces the area of impact with a soft object, e.g. a stress ball, and forms a dent around the impact area.
+In Blender, however, this simple deformation technique is absent for the most part.
+Of course, the objects could be simulated as part of a soft body or cloth simulation to get the desired outcome, but this technique is cumbersome and not easy to control.
+Complicated physics simulations are problematic in the context of rigged objects, and do not always produce the prefered results.
+The proposed solution would be a simple, one-time executable operation to provide a simple way to get the desired shape, and apply it e.g. as a shape key.
+Ideal would be the implementation as a C++-modifier for Blender in order to have it update automatically, but this out of the scope of the project for now.
+
+## Motivation
+
+Physics simulations for Blender provide realistic, but hard-to-control and not always viable methods for the deformation of soft objects.
+Particularly, a simple volume displacement which creates (semi-)realistic dents is not available, especially for rigged objects, which are problematic to use with physics simulations.
+When I was working with soft objects as part of an animation, I realised that Blender does not provide good tools for this kind of work.
+The problem can be solved via hand-sculpted shape keys, but creating those is very time consuming.
+Since this kind of deformation is actually present in a lot of daily interactions (considering hands themselves can act as a sort of soft body), it makes it important for ultra-realistic animation.
+The shadows created by these deformations are important for realism.
+I imagine others would be interested in this feature as well. 
```

Comment:
I found this idea interesting as I indeed had a need for creating such an effect in the past.